src/components/Main.jsx

```diff
@@ -5,13 +5,65 @@ import Fully from '../assets/icon-fully-customizable.svg?react';
 import { Hero } from './widget/Hero';
 import { Feature } from './widget/Feature';
 import { ShortenField } from './widget/ShortenField';
+import { useEffect, useState } from 'react';
+const myKey = import.meta.env.VITE_MY_SECRET_KEY;
 
 function Main() {
+  const [dataUrl, setDataUrl] = useState([]);
+  console.log(dataUrl);
+
+  async function shorteningUrl(longUrl) {
+    const endpoint = 'https://api.shorten.rest/aliases';
+    const data = {
+      aliasName: null,
+      destinations: [
+        {
+          url: longUrl,
+          country: null,
+          os: null,
+        },
+      ],
+    };
+    try {
+      const response = await fetch(endpoint, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+          'x-api-key': myKey,
+        },
+        body: JSON.stringify(data),
+      });
+      if (!response.ok) {
+        throw new Error(`Error: ${response.status}`);
+      }
+      const jsonResponse = await response.json();
+
+      const shortenedUrl = jsonResponse.shortUrl;
+
+      const newDataUrl = { originalUrl: longUrl, shortUrl: shortenedUrl };
+      localStorage.setItem('listOfUrl', JSON.stringify(newDataUrl));
```

Comment:
The name of an item is `listOfUrl`, so I would expect an array there. But nooo, you want an object there. Why?